src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+
+    public static Ball getRandomBall() {
+        Color color = ColorSupplier.getRandomColor();
+        int number = new Random().nextInt(MAX_NUMBER) + 1;
```

Comment:
color supplier and random should be class fields. You are creating a new random instance per each method call, while we can do it only once